src/components/MainPage/Exporters/SpreadsheetExporter.vue

```diff
@@ -61,24 +97,55 @@ export default {
     async submitForm() {
       this.loading = true;
       try {
-        this.saveExporterProperties({ name, properties: this.properties });
+        if (this.isNewSpreadsheet) {
+          await this.createNewSpreadsheet();
+        }
+
         const result = await saveTransactionsToGoogleSheets(
```

Comment:
what will happen if the spreadsheet exists but there isn't _isbd sheet on it?

Reply:
I forgot to test that. I created a new method to create a sheet if it not exist, right before the upload.